lib/python/qmk/util.py

```diff
@@ -3,12 +3,14 @@
 import contextlib
 import multiprocessing
 import sys
+import re
 
 from milc import cli
 
 maybe_exit_should_exit = True
 maybe_exit_reraise = False
 
+TRIPLET_PATTERN = re.compile(r'^(\d+)\.(\d+)\.(\d+)')
```

Comment:
Constants should not be after globals, and this should be moved above `maybe_exit_should_exit`.